### PR TITLE
Calculate Reward stub and convert functions for Qi and Quai

### PR DESCRIPTION
**Pull request by kiltsonfire**

### CODE REVIEW

File: consensus/misc/rewards.go [hunk 1]
```diff
@@ -3,12 +3,37 @@ package misc
...
+	qiPerQuai := new(big.Int).Div(calculateQiReward(currentHeader), calculateQuaiReward(currentHeader))
+	return new(big.Int).Mul(quaiAmt, qiPerQuai)
+}
+
+// CalculateQuaiReward calculates the quai that can be recieved for mining a block and returns value in qits
+func calculateQuaiReward(header *types.Header) *big.Int {
+	return common.Big1
+}
+
+// CalculateQiReward caculates the qi that can be received for mining a block and returns value in its
```
Comment (jdowning100) [inline consensus/misc/rewards.go:36]: Returns value in qits

File: consensus/misc/rewards.go [hunk 2]
```diff
@@ -3,12 +3,37 @@ package misc
...
+	return new(big.Int).Mul(qiAmt, quaiPerQi)
+}
+
+// Calculate the amount of Qi that Quai can be converted to. Expect the current Header and the Quai amount in "its", returns the Qi amount in "qits"
+func QuaiToQi(currentHeader *types.Header, quaiAmt *big.Int) *big.Int {
+	qiPerQuai := new(big.Int).Div(calculateQiReward(currentHeader), calculateQuaiReward(currentHeader))
+	return new(big.Int).Mul(quaiAmt, qiPerQuai)
+}
+
+// CalculateQuaiReward calculates the quai that can be recieved for mining a block and returns value in qits
```
Comment (jdowning100) [inline consensus/misc/rewards.go:31]: returns value in its

File: consensus/misc/rewards.go [hunk 3]
```diff
@@ -3,12 +3,37 @@ package misc
...
+func calculateQuaiReward(header *types.Header) *big.Int {
+	return common.Big1
+}
+
+// CalculateQiReward caculates the qi that can be received for mining a block and returns value in its
+func calculateQiReward(header *types.Header) *big.Int {
+	return common.Big1
```
Comment (jdowning100) [inline consensus/misc/rewards.go:38]: Can I request that the stub returns more? For testing. Maybe max denomination

**Review by Djadih — DISMISSED** [review]

File: consensus/misc/rewards.go
Comment (Djadih) [inline consensus/misc/rewards.go:1]: This file has merge conflicts to main

**Review by Djadih — CHANGES_REQUESTED** [review]

**Pull request merged**